ACMAS/app/ACMAS/settings.py

```diff
@@ -1,31 +1,33 @@
 """
 Django settings for ACMAS project.
 
-Generated by 'django-admin startproject' using Django 4.0.5.
+Generated by 'django-admin startproject' using Django 4.0.7.
 
 For more information on this file, see
 https://docs.djangoproject.com/en/4.0/topics/settings/
 
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/4.0/ref/settings/
 """
-
 import os
 from pathlib import Path
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-867+yl0pyu4h@^5cb&tl&j!&e^fh4-8bn97x*vp713*ujttvl("
+SECRET_KEY = os.environ.get("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = int(os.environ.get("DEBUG", default=0))
```

Comment:
Would this work `DEBUG = os.environ.get('DEBUG', default=False)`?
I would prefer bool over int for representing a boolean

Reply:
Resolved